pluma/Makefile.am

```diff
@@ -185,16 +191,16 @@ INTROSPECTION_GIRS = Pluma-1.0.gir
 INTROSPECTION_SCANNER_ARGS = -I$(top_srcdir) --warn-all
 
 Pluma-1.0.gir: pluma
-Pluma_1_0_gir_INCLUDES = Gtk-3.0 GtkSource-3.0
-Pluma_1_0_gir_FILES = $(INST_H_FILES) $(libpluma_c_files) $(BUILT_SOURCES)
+Pluma_1_0_gir_NAMESPACE = Pluma
 Pluma_1_0_gir_VERSION = 1.0
 Pluma_1_0_gir_PROGRAM = $(builddir)/pluma
-Pluma_1_0_gir_NAMESPACE = Pluma
+Pluma_1_0_gir_FILES = $(INST_H_FILES) $(libpluma_c_files) $(BUILT_SOURCES)
+Pluma_1_0_gir_INCLUDES = Gtk-3.0 GtkSource-3.0
 
-girdir = $(datadir)/gir-1.0
+girdir = $(datadir)/pluma/gir-1.0
```

Comment:
Why do you move the Pluma-1.0.gir file out of the $(datadir)/gir-1.0 folder?

Reply:
my mistake, done